packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -137,17 +121,7 @@ export default class TSGenerator extends CodeGeneratorLanguage {
 
     this.types = new Map();
     this.methodGenerics = new Map();
-    this.schemas = new Map();
-  }
-
-  static formatter(content: string) {
-    return prettier(content, {
-      format: true,
-      style: {
-        printWidth: 120,
-        singleQuote: true,
-      },
-    } as JSONSchemaToTypescriptOptions);
+    this.schemas = {};
```

Comment:
So I can do `JSON.stringify(schemas)` in `schemas.ts` and dump out all those schemas this is no longer a map because this is now being treated as a general object in order for me to set data into it with `lodash.setWIth()`.